Systems/carburetor.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+
+<!--
```

Comment:
Perhaps you could add a copyright block like we have in the other `Systems/c172p-*.xml` files? Also, we have (apparently) sort of a convention to prepend `c172p-` to files containing JSBsim systems in the Systems/ directory.